bundles/org.openhab.binding.epsonprojector/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="epsonprojector" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>Epson Projector Binding</name>
+	<description>This binding is compatible with Epson projectors which support the ESC/VP21 protocol</description>
+	<author>Pauli Anttila, Yannick Schaus</author>
```

Comment:
What about your name?

Reply:
Its getting crowded in there... How many authors is too much?

Reply:
If I understand correctly, you took over the binding and you are the maintainer now. So, I'd suggest that it's only your name in the author tag.

Reply:
I agree author != maintainer, which is why I omitted putting my name there.

Reply:
It looks like it is a moot point anyway: #9113

Reply:
Wouldn't be suprised if it was caused by this exchange ;)